package.json

```diff
@@ -221,7 +221,7 @@
       "json"
     ],
     "transform": {
-      ".(ts|tsx)": "ts-jest"
+      ".(jsx|js|ts|tsx)": "ts-jest"
```

Comment:
why do we need to add js in ts transformer?

Reply:
you need to `npm i brave-ui@latest` first to try this.

this line is needed so you can transform jsx/js files from brave-ui. without it tests still fail.

Reply:
btw ts-jest uses babel-jest under the hood. you need it so you can transform es6 modules to commonjs and make jest happy